uber/config.py

```diff
@@ -568,7 +571,10 @@ def PREREG_AGE_GROUP_OPTS(self):
     
     @property
     def NOW_OR_AT_CON(self):
-        return c.EPOCH.date() if date.today().date() <= c.EPOCH.date() else uber.utils.localized_now().date()
+        try:
+            return c.EPOCH.date() if date.today() <= c.EPOCH.date() else uber.utils.localized_now().date()
+        except:
+            traceback.print_exc()
```

Comment:
Is this functionally the same as log.error?

Reply:
Oops, this was some debugging left in. This line was throwing an AttributeError, but all AttributeErrors on Config just trigger the __getattr__ code, which in this case just raises a non-helpful AttributeError in return (https://github.com/magfest/ubersystem/blob/main/uber/config.py#L845)

Reply:
Oh yea, I see that a ton and it's super annoying. I don't suppose we can solve that in getattr, can we?

Reply:
We could use [__getattribute__](https://docs.python.org/3/reference/datamodel.html#object.__getattribute__) instead. That would get called on all accesses, then could default to falling through. I think that would let AttributeError's bubble up as normal.

Reply:
Yea, that looks reasonably straightforward!